src/index.html

```diff
@@ -3,11 +3,359 @@
+          <button class="carousel__button carousel__button--left" href="#">
+            <img src="images/slider/arrow-left.svg" alt="arrow-left">
+          </button>
+          <button class="carousel__button carousel__button--right" href="#">
+            <img src="images/slider/arrow-right.svg" alt="arrow-right">
+          </button>
+          <div class="carousel__button__wrapper">
+            <p>intro</p>
+            <span>By the same illusion which lifts the<br>horizon.</span>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="who-we-are" id="who-we-are">
+      <h3>Who we are</h3>
+      <p>We embrace a strategic approach to creative ideas. We are interested in people and human relationships. This is the main thing you need to know about us. We believe in the power of bold ideas that can solve business challenges.</p>
+    </section>
+
+    <section class="our-expertise__block">
+      <h2>Our Expertise</h2>
+      <div class="our-expertise__block__wrapper">
+        <div class="our-expertise__card">
+          <img src="images/what-we-do/what-we-do-1.jpg" alt="branding">
+          <h4>Branding</h4>
+          <p>We create additional value for companies, products, services as well as verbal and visual ways to deliver it to the audience.</p>
+        </div>
+        <div class="our-expertise__card">
+          <img src="images/what-we-do/what-we-do-2.jpg" alt="communication">
+          <h4>Communication</h4>
+          <p>We strive to create communications that can increase media performance. We use everything — words, meanings, stories, art, movies.</p>
+        </div>
+        <div class="our-expertise__card">
+          <img src="images/what-we-do/what-we-do-3.jpg" alt="strategy">
+          <h4>Strategy</h4>
+          <p>We create business growth strategies, from the moment of its birth to the achievement of the necessary business indicators.</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="process__block" id="services">
+      <div class="process__top">
+        <p>Services</p>
+        <h2>Air is a full service creative agency</h2>
+        <span>Deep analytics, strong strategy and bright creative ideas.
+          <br><br> We are sure that first-rate job is possible only if all three components are united.
+        </span>
+      </div>
+      <div class="process__card__wrapper">
+        <div class="process__card">
+          <span>001</span>
+          <div class="process__card--categories">
+            <span>Brand Development</span>
+            <span>Copywriting</span>
+            <span>Logo & Website Design</span>
+            <span>Packaging</span>
+          </div>
+          <div class="process__card-btn--wrapper">
+            <button class="process__card-btn">
+              <a href="#services">
+                <img src="images/arrow-forward.svg" alt="arrow-forward">
+              </a>
+            </button>
+            <h4>Learn more</h4>
```

Comment:
Learn more should be a link to work in consonance with the arrow to redirect the user if the text is clicked [Checklist #14](https://github.com/mate-academy/layout_dia#:~:text=Inside%20the%20cards%20in%20PROCESS%20sections%2C%20make%20a%20link%20out%20of%20the%20%22Learn%20More%22%20text%2C%20so%20that%20it%20hovers%20simultaneously%20with%20the%20arrow%20button%20on%20the%20left)